Misc/NEWS.d/next/Core and Builtins/2022-03-07-11-51-51.bpo-46942.57obVi.rst

```diff
@@ -0,0 +1,4 @@
+:func:`idlelib.calltip.get_argspec` now correctly reports that
```

Comment:
This is an odd way to put it. It's not idlelib that changed, but classobject.c. Can we just say something like "Use Argument Clinic for the types.MethodType constructor"?